content/2019/2019-11/2019-11-01-pulsar-weekly.md

```diff
@@ -127,14 +127,14 @@ This is the weekly community update for 2019-10-26 ~ 2019-11-01, with updates on
     https://www.nuget.org/packages/Pulsar.Client/
     
 
-### Event / News
+### Event/News
```

Comment:
Many people wonder whether to use a space before and after slashes in a sentence. The answer is typically no on both fronts. Spaces before slashes should be avoided. The only time it’s acceptable to use a space after a slash is when breaking up lines of a poem, song, or play. 

Example
Mary had a little lamb/ little lamb, little lamb/ Mary had a little lamb/ whose fleece was white as snow.